e2e/src/context.ts

```diff
@@ -28,6 +28,23 @@ export interface Context {
   sessionIds?: string[];
 }
 
+export const contextDefaults = {
+  user: {
+    firstName: "Test",
+    lastName: "Test",
+    // Gov.uk Notify requests testing service use smoke test email addresses
+    // see https://docs.notifications.service.gov.uk/rest-api.html#smoke-testing
+    email: "simulate-delivered@notifications.service.gov.uk",
```

Comment:
I noticed this this morning looking into the email submission audit table - good to have this in place, should have spotted this sooner sorry!

![Screenshot 2023-05-23 at 10 40 24](https://github.com/theopensystemslab/planx-new/assets/20502206/f12e3be0-d749-4699-919e-65d87864e83c)